### PR TITLE
Add layout item

### PR DESCRIPTION
**Pull request by gorgeousvlad**

### CODE REVIEW

File: src/sub-blocks/LayoutItem/utils.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import {LayoutItemProps, LinkTheme, MediaProps} from '../../models';
+
+export const getLayoutItemLinks = (links: LayoutItemProps['content']['links']) =>
+    links?.map((link) => ({
+        theme: 'normal' as LinkTheme,
+        ...link,
+    }));
+
+export const hasFullScreen = ({dataLens, image}: MediaProps) => {
+    // datalens and slider media card don't support fullscrPcareen
```
Comment (NikitaCG) [inline src/sub-blocks/LayoutItem/utils.ts:10]: small typo in word fullScreen

File: src/components/VideoBlock/VideoBlock.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ const VideoBlock = (props: VideoBlockProps) => {
     const [hidePreview, setHidePreview] = useState(false);
     const norender = (!stream && !record) || !src;
     const [currentHeight, setCurrentHeight] = useState(height || undefined);
-    const fullId = `${iframeId}-${id || src}`;
+    const fullId = `${iframeId}-${id || src}${fullScreen ? '-fullscreen' : ''}`;
```
Comment (NikitaCG) [inline src/components/VideoBlock/VideoBlock.tsx:68]: can we use library like this - https://www.npmjs.com/package/uuid for create unique id ?

File: src/sub-blocks/MediaCard/MediaCard.tsx [hunk 3]
```diff
@@ -2,44 +2,20 @@ import React from 'react';
...
-    // datalens and slider media card don't support fullscreen
-    return !(dataLens || Array.isArray(image));
-};
-const CONTENT_SIZE = 's';
-
-const MediaCard = ({border, fullScreen, metaInfo, content, ...mediaProps}: MediaCardProps) => (
-    <div className={b({hasContent: Boolean(content || metaInfo)})}>
-        <CardBase bodyClassName={b('body')} border={border}>
+const MediaCard = ({border, ...mediaProps}: MediaCardProps) => {
+    return (
```
Comment (yuberdysheva) [inline src/sub-blocks/MediaCard/MediaCard.tsx:12]: Use without `{` and `return`

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/components/VideoBlock/VideoBlock.tsx (hunk 2 above)
Comment (gorgeousvlad) [reply src/components/VideoBlock/VideoBlock.tsx:68]: i think yes, already, before there was a comparsion: has the video src changed or not in new compoent

**Review by yuberdysheva — APPROVED** [review]

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**